src/utils/parse.ts

```diff
@@ -1,4 +1,4 @@
-import type { InvalidDataReason } from "../errors";
+import type { InvalidDataReason } from "../errors/index";
```

Comment:
`flowgen` doesn't work well when implicitly including the `index` file so I just made it explicit here.

Reply:
can you clarify what kind of problem does implicitly including the index file cause? I tried reverting this line and the outcome of build:flowtypes didn't seem to change

Reply:
It causes the type of `InvalidDataReason` in the Flow code to not be imported as a type, causing a Flow error in the parse.flow

Reply:
I see, wondering if there's some automated way in which we could identify such broken flow types so we know and can fix it. I tried checking the generated files with flow-bin but that didn't seem to catch it. Maybe you can suggest something we can incorporate in our CI pipeline

Reply:
Okay, I added checking the Flow types as a build step when publishing. You can disable it in the future if you decide you don't want to maintain it at some point. You'll notice if you remove the `/index`, Flow will give you an error in the generated code as I mentioned